lightning/src/onion_message/messenger.rs

```diff
@@ -1413,11 +1424,11 @@ where
 	fn handle_onion_message(&self, peer_node_id: &PublicKey, msg: &OnionMessage) {
 		let logger = WithContext::from(&self.logger, Some(*peer_node_id), None, None);
 		match self.peel_onion_message(msg) {
-			Ok(PeeledOnion::Receive(message, path_id, reply_path)) => {
+			Ok(PeeledOnion::Receive(message, _tlvs, reply_path)) => {
```

Comment:
Let's call this `_recipient_data`

Reply:
Alright! [653636b](https://github.com/lightningdevkit/rust-lightning/commit/653636b521ec0d4dba73b7aff83e80d594928a9e)